apps/dashboard/src/contract-ui/tabs/overview/components/LatestEvents.tsx

```diff
@@ -171,14 +170,7 @@ const EventsFeedItem: React.FC<EventsFeedItemProps> = ({ transaction }) => {
                 bg="transparent"
                 onClick={() => {
                   onCopy();
-                  toast({
-                    variant: "solid",
-                    position: "bottom",
-                    title: "Transaction hash copied.",
-                    status: "success",
-                    duration: 5000,
-                    isClosable: true,
-                  });
+                  toast("Transaction hash copied.");
```

Comment:
This shouldn't be "success" variant - if there is other visual feedback for copy action - we can completely drop this - otherwise use "info" variant